azkaban-web-server/src/web/js/azkaban/view/schedule-panel.js

```diff
@@ -217,6 +195,9 @@ function updateOutput() {
 
 function updateExpression() {
   $('#nextRecurId').html("");
+
+  //Unix Cron use 0-6 as Sun--Sat, but Quartz use 1-7. Due to later.js only supporting Unix Cron, we have to make this transition.
+  //transformFromQuartzToUnixCron is defined in util/date.js
```

Comment:
I intended to place it here, in order to let readers know why we need this transition. What do you think?